web/libs/datamanager/src/components/MainView/GridView/ImagePreview.tsx

```diff
@@ -195,7 +196,6 @@ const ImagePreview = observer(({ task, field }: ImagePreviewProps) => {
     ? {
         maxWidth: "100%",
         maxHeight: "100%",
-        transform: `translate(${offset.x}px, ${offset.y}px) scale(${scale})`,
```

Comment:
hmmm, why do you remove this line? it's crucial for images being interactive in this preview